### ISSUE TITLE
HVV ferry departures not working.

### ISSUE THREAD

**Issue by fxb** [public-transport/hafas-client#261]
It seems like HVV ferry departures are not working.

For example id `6256` is the `Teufelsbrück` stop.

Requesting departures via `https://v5.hvv.transport.rest/stops/6256/departures?ferry=true` always returns an empty array, while the [HVV website](https://www.hvv.de/de/fahrplaene/abfahrten) lists ferry departures.

**Comment by derhuerst** [comment]
I'll check with `hafas-client`'s debugging CLI:

```shell
cd hafas-client
./tools/debug-cli/cli.js hvv locations 'Teufelsbrück' | jq
```

```js
[
	{
		"type": "stop",
		"id": "6256",
		"name": "Teufelsbrück",
		"location": {
			"type": "location",
			"id": "6256",
			"latitude": 53.54644,
			"longitude": 9.862866
		},
		"products": {
			"subway": false,
			"suburban": false,
			"akn": false,
			"regional-express-train": false,
			"regional-train": false,
			"ferry": true,
			"bus": false,
			"express-bus": false,
			"anruf-sammel-taxi": false,
			"long-distance-train": false,
			"long-distance-bus": false
		}
	},
	{
		"type": "stop",
		"id": "5822",
		"name": "Teufelsbrück (Fähre)",
		"location": {
			"type": "location",
			"id": "5822",
			"latitude": 53.547627,
			"longitude": 9.863845
		},
		"products": {
			"subway": false,
			"suburban": false,
			"akn": false,
			"regional-express-train": false,
			"regional-train": false,
			"ferry": false,
			"bus": true,
			"express-bus": false,
			"anruf-sammel-taxi": false,
			"long-distance-train": false,
			"long-distance-bus": false
		}
	},
	// …
]
```

```shell
./tools/debug-cli/cli.js hvv departures 6256 '{duration: 600}' | jq
# []
./tools/debug-cli/cli.js hvv departures 5822 '{duration: 600}' | jq
# []
```

TLDR: Yes, it seems like **HVV's HAFAS `mgate.exe` API doesn't return ferries**.

**Comment by derhuerst** [comment]
Can you check with other ferry lines and at another stop?

**Comment by fxb** [comment]
I did check some other stops as well, and none seemed to return anything.

**Comment by fxb** [comment]
Another example that basically only has ferrys stopping is `3598` ("Rüschpark")

**Comment by fxb** [comment]
I haven't read up on the internal workings of things yet, but could it have something to do with the bitmasks defined in https://github.com/public-transport/hafas-client/blob/master/p/hvv/products.js ? 

I reported another issue that just from my gut feeling could also be caused by some mismatch of these bitmasks? See: https://github.com/public-transport/hafas-client/issues/262

**Comment by derhuerst** [comment]
> [...] could it have something to do with the bitmasks [...] ?

Yes, this is how `hafas-client` requests departures for specific "products" (basically like categories).

But even without [the bitmask in place](https://github.com/public-transport/hafas-client/blob/cbc7f63dcc6a1cda3271966facc93508bd46af54/format/station-board-req.js#L7), thus using the server's default, it doesn't return any departures for the stops `6256` & `5822`.
